tests/test_product_requirements_text_QCI97.py

```diff
@@ -0,0 +1,77 @@
+# QCI-97
+
+# In the "Select products to deploy screen, there is a text box to the right
+# of the products list.   It as you select various products will contain text
+# that explains what is required by each product selected.   Be sure the text
+# is correct for the products selected.
+
+def test_product_requirements_text(new_deployment_pg):
+    general_expected_text = (
+        "General\nRed Hat Customer Portal username and password. This account"
+        " must be an Organization Administrator within the Portal\nSufficient"
+        " subscriptions to complete this deployment\nHost machines are started"
+        " and discoverable\nNetworks and subsets are set up and available"
+    )
+    rhv_expected_text = "\n".join([
```

Comment:
'rhv_expected_text' is an example of lines joined with '\n'. I have left it here for you to consider, what is better. I don't particularly like this approach, nor do I like the other one, where the text is saved as an one-liner. At this point I have no idea what is more readable. Both of these are working though.